contents/blog/posthog-vs-statsig.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: "In-depth: PostHog vs Statsig"
+date: 2023-12-06
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: General
+tags:
+    - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+PostHog and Statsig both offer A/B testing and feature flags, but they're different in two important ways:
+
+- **Statsig** offers advance experimentation tools, such as multi-armed bandit and holdout testing. It's ideal for data scientists, product managers, and growth teams who need to conduct complex A/B testing.
+
+- **PostHog** is an all-in-one platform built to help engineers understand users and create better products. Beyond [feature flags](/feature-flags) and [A/B testing](/ab-testing), it includes [product analytics](/docs/product-analytics), [session replay](/docs/session-replay), [surveys](/docs/surveys), and more.
+
+This post compares their platforms, experimentation features, pricing, and more.
+
+## How is PostHog different?
+
+### 1. It is an all-in-one platform
+
+PostHog brings together all the tools engineers need for testing, releasing, and measuring the success of new features. Feature flags and A/B testing are only part of a suite of tools PostHog offers.
+
+![PostHog homepage features](../images/blog/posthog-vs-statsig/all.png)
+
+PostHog combines usage, performance, and behavioral data with flags and experiments. PostHog’s [data warehouse](/docs/data-warehouse) also enables you to pull in data from external sources.
```

Comment:
I don't know if I'd mention data warehouse yet, we don't really know how well it will work with A/B testing and feature flags.

Reply:
@EDsCODE any thoughts?

Reply:
@ivanagas FYI this is on the growthbook blog https://posthog.com/blog/posthog-vs-growthbook

Reply:
Hmm not sure where my other comment went. must've not pressed send earlier. 

To be safe, I would agree with @ivanagas and avoid drawing connections between data warehouse and experimentation for now. We're still in an experimental/validation phase with connecting external data